aplib.net-demo/Assets/Scripts/Entities/Player/MouseLock.cs

```diff
@@ -14,25 +19,42 @@ private void Awake()
     }
 
     /// <summary>
-    /// Shows the mouse and unlocks it.
+    /// Shows the mouse and unlocks it. Useful on websites.
+    /// </summary>
+    public void OnShowMousePressed() => EnableMouseCursor();
+
+    /// <summary>
+    /// On left mouse click, go back into the game and lock the cursor.
     /// </summary>
-    public void OnShowMousePressed()
+    public void OnLeftMousePressed()
+    {
+        if (CanvasManager.IsCursurNeeded) // if cursor is needed, enable cursor
+        {
+            EnableMouseCursor();
+        }
+        else if (_showMouse && !CanvasManager.IsCursurNeeded) // if cursor is showing and is not needed, disable
+        {
+            DisableMouseCursor();
+        }
```

Comment:
Volgens mij moet het dit zijn maar het is mogelijk dat ik de editorconfig van verschillende projecten door elkaar haal.
```suggestion
        if (CanvasManager.IsCursurNeeded) // if cursor is needed, enable cursor
            EnableMouseCursor();
        else if (_showMouse && !CanvasManager.IsCursurNeeded) // if cursor is showing and is not needed, disable
            DisableMouseCursor();
```